examples/simple/src/index.tsx

```diff
@@ -23,25 +23,43 @@ render(
             i18nProvider={i18nProvider}
             title="Example Admin"
             layout={Layout}
-            customRoutes={[
-                <RouteWithoutLayout
-                    exact
+        >
+            <CustomRoutes noLayout>
```

Comment:
Running the `simple` project, If you :

1. Remove the `noLayout` prop here
2. Logout
3. At the Login page, manually add `/custom` in the browser's address bar

Result: You get access to the project.

Is this expected?

Reply:
Yes because this route is not protected automatically. I may had some stories to showcase the possibilities, including protected routes without layout